gateway/api/v1/views.py

```diff
@@ -18,8 +18,8 @@ class ProgramViewSet(views.ProgramViewSet):  # pylint: disable=too-many-ancestor
     Quantum function view set first version. Use ProgramSerializer V1.
     """
 
-    queryset = Program.objects.all()
```

Comment:
It doesn't appear this variable was used after it was defined.... was that some kind of Django config?

Reply:
yes, it is a default django viewset attribute that specified what needs to be returned. It can be overwritten by `get_queryset` which is exactly doing later in this PR :)

Reply:
yep, exactly that!